tests/e2e/tests/generate/generate-util.ts

```diff
@@ -0,0 +1,32 @@
+import { ng } from '../../utils/process';
+import { expectFileToExist } from '../../utils/fs';
+import { expectToFail } from "../../utils/utils";
+
+export interface TestGenerateParam {
+  blueprint: 'component' | 'directive' | 'guard' |
+    'module' | 'pipe' | 'service';
+  name: string;
+  flags: string[];
+  pathsToVerify: string[];
+}
+
+export function testGenerate(params: TestGenerateParam): Promise<any> {
+  return ng('generate', params.blueprint, params.name, ...params.flags)
```

Comment:
Could you do `const { blueprint, name, flags, pathsToVerify } = params`? It would simplify the code.